src/components/ProductCard/ProductCart.tsx

```diff
@@ -1,14 +1,48 @@
+import styles from './productCart.module.css';
+import utils from '../../styles/modules/utils.module.css';
+import SelectedProduct from '../SelectedProduct/SelectedProduct';
+import StarIcon from '../Icon/StarIcon';
+
 interface ProductCardProps {
   image: string;
   title: string;
   price: number;
   description: string;
   rating: number;
-  isActive: boolean;
+  ratingNumbers: number;
 }
 
-const ProductCard = (_: ProductCardProps) => {
-  return <></>;
+const ProductCard = ({
+  image,
+  title,
+  price,
+  description,
+  rating,
+  ratingNumbers,
+}: ProductCardProps) => {
+  return (
+    <div className={styles.product}>
+      <div className={`${styles.imageGroup} ${utils.relative}`}>
+        <img className={styles.productImage} src={image} alt={title} />
```

Comment:
What happen if source image provided is wrong, will it break the layout

Reply:
It not break the layout

![image](https://github.com/user-attachments/assets/09fa14ad-b063-46bd-a99d-77077088fce2)